workshops/forms.py

```diff
@@ -1,15 +1,16 @@
 import re
+from django.core.exceptions import ValidationError
 
 from django import forms
 from django.core.validators import RegexValidator
 from django.forms import (
     HiddenInput, CheckboxSelectMultiple, TextInput, modelformset_factory,
     RadioSelect,
-)
+    URLField)
```

Comment:
We keep `)` on a separate line to not trigger line changes when new imports are added. Please use this format.